Tools/scripts/uploader.py

```diff
@@ -78,7 +79,7 @@
                   '/dev/serial/by-id/usb-Holybro*',
                   '/dev/tty.usbmodem*']
 
-if "cygwin" in _platform:
+if "cygwin" in _platform or (os.path.exists("/proc/version") and "Microsoft" in subprocess.check_output(["cat", "/proc/version"])):
```

Comment:
I think a more phythonic way (and safer) would be :
````
import platform
"Microsoft" in platform.uname()[2]
````

On ubuntu 18.04 : 
````
>>> platform.uname()
('Linux', 'AZDNUX0003', '4.15.0-46-generic', '#49-Ubuntu SMP Wed Feb 6 09:33:07 UTC 2019', 'x86_64', 'x86_64')
````

On WSL:
````
>>> platform.uname()
('Linux', 'AZDDST0018', '4.4.0-17763-Microsoft', '#253-Microsoft Mon Dec 31 17:49:00 PST 2018', 'x86_64', 'x86_64') 
````

Reply:
Hmm.. I thought I looked at that. I'll check again

Reply:
yup, platform.uname()[2] looks good. Let's go with that! I've force pushed a fix